src/settings-compat-v7/_components.scss

```diff
@@ -131,6 +131,7 @@ $c-breadcrumb-link: $c-primary !default;
 $c-breadcrumb: $c-text-base !default;
 $m-breadcrumb-items: $m-h-small !default;
 $size-breadcrumb-icon: 18px !default;
+$td-breadcrumb-link-hover: none !default;
```

Comment:
Would it be much asking if you could be so kind to move all these variables into the component and remove breadcrumb's settings-compat-v7 dependency?

Reply:
Sure no problem, I will move all these breadcrumb variables inside components/_breadcrumb.scss  :)

Reply:
Thx! but you should move them to https://github.com/SUI-Components/sui-components/blob/master/components/molecule/breadcrumb/src/index.scss and remove the first line

Reply:
@jcger removed, take a look here please --> https://github.com/SUI-Components/sui-components/pull/546/files